samples/apps/smallbank/smallbank.cmake

```diff
@@ -47,4 +48,21 @@ if(BUILD_TESTS)
       --max-writes-ahead 1 --warmup 1000 --cooldown 1000
       --metrics-file small_bank_wc_metrics.json
   )
+
+  # It is better to run performance tests with forwarding on different machines
```

Comment:
Do we still need the warmup/cooldown then?

Reply:
Let's take it out in a separate PR as discussed.

Reply:
I'll address this in another PR.